### ISSUE TITLE
2024.10.22-beta - Missing manifest.json

### ISSUE THREAD

**Issue by Nerwyn** [beecho01/material-symbols#10]
**Describe the bug**
The repo is missing a [manifest.json](https://www.hacs.xyz/docs/publish/integration/#manifestjson) as required by HACS, making it so that it cannot be installed. This also prevents the repository from being added as a custom repository for v1.0.0.

**To Reproduce**
1. Follow the README instructions to uninstall v1.0.0 and restart Home Assistant.
2. Try installing the 2024.10.22-beta. Note the unknown error.

At this point material symbols disappeared from my custom repositories list, and when I tried adding it back I got this error:

```
No manifest.json file found 'manifest.json'
{
    "action": "add_repository",
    "exception": "HacsException",
    "message": "No manifest.json file found 'manifest.json'"
}
```

**Expected behavior**
Repository structure should follow the [HACS integration documentation](https://www.hacs.xyz/docs/publish/integration/).

**Screenshots**
![image](https://github.com/user-attachments/assets/ce41eb89-ecec-4e96-9307-084265c4a60d)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Brave
 - Version: 1.71.114

**Additional context**
I got excited to try the new version, but now I can't even install v1.0.0 as HACS won't let me add the repository as an integration or dashboard add on as 2024.10.22-beta is not compliant.

**Comment by beecho01** [comment]
Hi @Nerwyn, 

This is unusual as the file is in the location expected (unless I have missed something). Let me look into this and attempt a fix asap.

**Comment by beecho01** [comment]
@Nerwyn I have removed release 2024.10.22-beta so that v1.0.0 should be usable now as a "Dashboard" repository again.

**Comment by Nerwyn** [comment]
Thanks! I'm able to reinstall it now.

That's weird, maybe it was a HACS bug on my end. What if you try recreating the beta but not setting it as the latest release?

**Comment by beecho01** [comment]
@Nerwyn I took your advice and did just that. The integration update is now a pre-release.

I have reached out to the HACS discord community as I am stumped on why this isn't working. There are suggestions that the `manifest.json` encoding (not UTF-8) might be a cause, but I have checked and it doesn't appear to be the case.

Would you check if manually installing works and this is solely a HACS issue?

**Comment by beecho01** [comment]
This has now appears to been resolved in [2024.10.23-beta](https://github.com/beecho01/material-symbols/releases/tag/2024.10.23-beta)

**Comment by Nerwyn** [comment]
It's working for me now! Thanks!
